### DISCUSSION TITLE
Find if Thread is "newly_created"

### DISCUSSION THREAD

**Discussion by civilCornball** [discord-php/DiscordPHP#1242] [Q&A] [answered]
I'm telling my bot to look for the THREAD_CREATE event, and send a message there.

But, because it's sending a message, it adds itself to the thread, firing the THREAD_CREATE event again.

I found this in the Discord documentation: https://discord.com/developers/docs/topics/gateway-events#thread-create

But I don't know how to access it, and `var_dump()`-ing the $thread object tells me that there's never any "newly_created" string within the returned object.

"$thread->created" also doesn't work, for some godforsaken reason, leaving me stuck.

I'm on DiscordPHP 10.0.0-RC6.

**Comment by SQKo** [comment, marked as answer]
Added `$thread->newly_created` attribute in latest [dev-master#28409d4](https://github.com/discord-php/DiscordPHP/commit/28409d48f4b566a113ff1cb03015dc009da892d5)

Note this attribute may only exist inside THREAD_CREATE event and is nullable.

Thanks for the heads up!
